Makefile

```diff
@@ -164,3 +165,96 @@ ifneq ($(HAS_WEBAPP),)
 	rm -fr webapp/node_modules
 endif
 	rm -fr build/bin/
+
+# server-debug builds and deploys a debug version of the plugin for your architecture.
+# Then resets the plugin to pick up the changes.
+.PHONY: debug
+debug: server-debug reset
+
+.PHONY: server-debug
+server-debug: server/.depensure
+
+	./build/bin/manifest apply
+	mkdir -p server/dist
+
+ifeq ($(OS),OSX)
+	cd server && env GOOS=darwin GOARCH=amd64 $(GO) build -gcflags "all=-N -l" -o dist/plugin-darwin-amd64;
+else ifeq ($(OS),Linux)
```

Comment:
Can we make this automatic for our two most common dev platforms? We should be able to use `uname` to detect between those.

Reply:
Good call, I forgot about uname.